index.html

```diff
@@ -0,0 +1,370 @@
+            <li>Back End Dev</li>
+            <li><img src="./images/Counter.png" alt="counter" /></li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+        </div>
+        <button>see project</button>
+      </div>
+      <div class="post">
+        <div class="postprofile">
+          <img src="./images/SnapshootPortfolio4.png" alt="" />
+        </div>
+        <div class="postcontent">
+          <h1>Multi-Post stories</h1>
+          <ul>
+            <li>CANOPY</li>
+            <li><img src="./images/Counter.png" alt="counter" /></li>
+            <li>Back End Dev</li>
+            <li><img src="./images/Counter.png" alt="counter" /></li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+        </div>
+        <button>see project</button>
+      </div>
+      <div class="post">
+        <div class="postprofile">
+          <img src="./images/SnapshootPortfolio0.png" alt="" />
+        </div>
+        <div class="postcontent">
+          <h1>Tonic</h1>
+          <ul>
+            <li>CANOPY</li>
+            <li><img src="./images/Counter.png" alt="counter" /></li>
+            <li>Back End Dev</li>
+            <li><img src="./images/Counter.png" alt="counter" /></li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+        </div>
+        <button>see project</button>
+      </div>
+      <div class="post">
+        <div class="postprofile">
+          <img src="./images/SnapshootPortfolio.png" alt="" />
+        </div>
+        <div class="postcontent">
+          <h1>Multi-Post stories</h1>
+          <ul>
+            <li>CANOPY</li>
+            <li><img src="./images/Counter.png" alt="counter" /></li>
+            <li>Back End Dev</li>
+            <li><img src="./images/Counter.png" alt="counter" /></li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+        </div>
+        <button>see project</button>
+      </div>
```

Comment:
- [ ] Let's change these four divs into one `ul` element containing four `li` elements, `div` should be used to separate unrelated content and a mixture of uls and lis would a friendly way to separate related ones.